### PR TITLE
Fix the copyright to reference the JSTransformers Team

### PR DESCRIPTION
**Pull request by RobLoach**
https://github.com/jstransformers/jstransformer/issues/9

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # The MIT License (MIT)
 
-Copyright (c) 2015 Your Name
+Copyright (c) 2015 [JSTransformers](http://github.com/jstransformers)
```
Comment (ForbesLindesay) [inline LICENSE.md]: Should this also be http://github.com/orgs/jstransformers/people ?

**Comment by TimothyGu** [comment]
LGTM.

**Comment by RobLoach** [comment]
@ForbesLindesay @TimothyGu @tunnckoCore This look good with you guys?

**Comment by tunnckoCore** [comment]
O yea! LGTM :)

**Pull request merged**